app/views/sections/show.html.slim

```diff
@@ -24,28 +31,44 @@
           - @latest_videos.each do |video|
             .medium-4.cell
               = link_to(video)
```

Comment:
For markup, avoid parenthesis and prefer `link_to video`.

Also, I think the `link_to` should be inside `- if video`, right?